### PR TITLE
docs(sidebars): create new-structured docs

### PR DESCRIPTION
**Pull request by JafarAz**
Required for merge:
- [ ] `pr-workflow-check / draft-release-check` is ✅ success
- Other rules GitHub shows you, or can be read in [configuration](../terraform/github.com/branches.tf) 

Makes review faster:
- [ ] PR title is my best effort to provide summary of changes and has clear text to be part of release notes 
- [ ] I marked PR by `misc` label if it should not be in release notes
- [ ] Linked Zenhub/Github/Slack/etc reference if one exists
- [ ] I was clear on what type of deployment required to release my changes (node, runtime, contract, indexer, on chain operation, frontend, infrastructure) if any in PR title or description
- [ ] Added reviewer into `Reviewers`
- [ ] I tagged(`@`) or used other form of notification of one person who I think can handle best review of this PR
- [ ] I have proved that PR has no general regressions of relevant features and processes required to release into production
- [ ] Any dependency updates made, was done according guides from relevant dependency
- Clicking all checkboxes 
- Adding detailed description of changes when it feels appropriate (for example when PR is big)

### CODE REVIEW

**Review by dzmitry-lahoda — APPROVED** [review]

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -65,6 +65,26 @@ const config = {
 					src: 'img/logo.svg',
 				},
 				items: [
+					{
+						to: '/parachains/picasso-parachain-overview',
+						position: 'left',
+						label: 'Networks',
+					  },
+					{
+						to: '/products/centauri/Dotsama-ibc',
```
Comment (0xBrainjar2) [inline docs/docusaurus.config.js:74]: should not just be dotsama IBC - cross ecosystem IBC 

And then list specifically the progress of each ecosystem  - format should be: 

-architecture 
-integrations 
-etc

File: docs/sidebars.js [hunk 2]
```diff
@@ -17,409 +17,349 @@ const isProd = process.env.NODE_ENV === 'production';
...
-                    items: [
-                        'products/cosmwasm/existing-cosmwasm-projects-to-deploy-on-ccw-vm',
-                        'products/cosmwasm/syngery-with-centauri-and-xcvm',
-                        'products/cosmwasm/writing-smart-contracts-with-cosmwasm',
-                    ],
-                },
-                {
-                    type: 'category',
-                    label: 'Apollo',
-                    link: {
```
Comment (0xBrainjar2) [inline docs/sidebars.js:155]: apollo + collators can be under picasso 

same with cosm wasm

File: docs/sidebars.js [hunk 3]
```diff
@@ -17,409 +17,349 @@ const isProd = process.env.NODE_ENV === 'production';
...
+        ],
+      },
+  
+      "parachains/centauri-chain",
+    ],
+  
+    centauri: [
+      "products/centauri/Dotsama-ibc",
+      "products/centauri/near-ibc-bridge",
+      "products/centauri/hyperspace-relayer",
```
Comment (0xBrainjar2) [inline docs/sidebars.js:159]: relayer can be its own product

File: docs/sidebars.js [hunk 4]
```diff
@@ -17,409 +17,349 @@ const isProd = process.env.NODE_ENV === 'production';
...
+      "products/centauri/light-clients",
+      "products/centauri/merkle-mountain-ranges",
```
Comment (0xBrainjar2) [inline docs/sidebars.js:161]: MMR + light clients + beefy ibc light client etc can all be under architecture within the first category

File: docs/sidebars.js [hunk 5]
```diff
@@ -17,409 +17,349 @@ const isProd = process.env.NODE_ENV === 'production';
...
+            },
             collapsible: true,
             collapsed: true,
-            items: [
-                'faqs/disclaimers-disclosures-for-composable-tokens',
-                'faqs/risk-factors',
-                'faqs/terms-of-use',
-            ]
-        }
+            items: ["products/xcvm/use-cases/swap"],
```
Comment (0xBrainjar2) [inline docs/sidebars.js:187]: XCVM should be composable VM

**Pull request closed without merging**